app/grandchallenge/products/views.py

```diff
@@ -241,10 +260,28 @@ def get_success_url(self):
         return reverse("products:product-list")
 
 
-class ProjectAirForm(PermissionRequiredMixin, CreateView):
-    template_name = "products/projectairfiles_form.html"
-    form_class = ProjectAirFilesForm
-    permission_required = f"{ProjectAirFiles._meta.app_label}.add_{ProjectAirFiles._meta.model_name}"
+class ProductsPostCreate(PostCreate):
+    template_name = "products/post_form.html"
 
-    def get_success_url(self):
-        return reverse("products:project-air")
+
+class ProductsPostList(PostList):
+    template_name = "products/post_list.html"
+    queryset = PostList.model.objects.filter(
+        published=True, tags__name__contains="Products"
+    )
+
+
+class ProductsPostDetail(PostDetail):
+    template_name = "products/post_detail.html"
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        obj = context["object"]
+        is_news = obj.tags.all().filter(name="News").exists()
```

Comment:
Filtering on `name` is fragile, use `slug` instead with the appropriate value.

Reply:
fixed